src/components/skjema-tilgang-sync.tsx

```diff
@@ -0,0 +1,37 @@
+import { useSkjemaTilgangStore } from '../stores/skjema-tilgang-store';
+import { useEffect } from 'react';
+import { hasFinishedWithData } from '../rest/utils';
+import { finnUtkast } from '../utils';
+import { useFetchStore } from '../stores/fetch-store';
+
+
+export function SkjemaTilgangSync() {
+
+    const { setErInnloggetAnsvarligVeileder, setErInnloggetVeilederBeslutter, setReadOnly } = useSkjemaTilgangStore();
```

Comment:
God ide å flytte ut skjema tilgang til en egen store 😄 
Det hadde kanskje vært greit å flytte `setErInnloggetAnsvarligVeileder` og  `setErInnloggetVeilederBeslutter`  til en egen store som f.eks useInnloggetVeilederStore(). Grunnen til dette er at hvem som er innlogget er greit å vite i andre deler i appen, og ikke bare for skjemaet.

Reply:
Jeg heller foretrekker å rename skjema-tilgang-store.ts til veileder-tilgang-store.ts og 
og skjema-tilgang-sync.tsx til veileder-tilgang-sync.tsx eller vedtak -tilgang-store.ts og vedtak-tilgang-sync.tsx. Hva synes du? Hvilke passer best?